packages/date/src/DateRange.js

```diff
@@ -296,6 +296,7 @@ const DateRange = ({
         <DateRangePicker
           renderMonthElement={renderMonthElement}
           minimumNights={0}
+          ariaDescribedBy={`${name.toLowerCase()}-feedback`}
```

Comment:
I think we should add this to the `DateRangeField` component instead. We can pass in `ariaDescribedBy={`${name.toLowerCase()}-feedback`}` to the `DateRange` component`. Then we can pass that `ariaDesribedBy` prop to the `DateRangePicker`